platform/frontend_connectors/schema_transformer_test.go

```diff
@@ -1600,3 +1586,247 @@ func Test_mapKeys(t *testing.T) {
 	}
 
 }
+
+func Test_acceptIntsAsTimestamps(t *testing.T) {
+	schemaTable := schema.Table{
+		Columns: map[string]schema.Column{
+			"@timestamp":   {Name: "@timestamp", Type: "DateTime64"},
+			"timestampInt": {Name: "timestampInt", Type: "UInt64"},
```

Comment:
As we discussed, I think this should be `DateTime`, however it would be good to check how elasticsearch behaves in this case.

Reply:
I'll check that